src/team/schema/team.schema.ts

```diff
@@ -0,0 +1,34 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+
+
+@Schema()
+export class Team {
+  @Prop({ unique: true, required: true })
+  id: number;
+
+  @Prop({ required: true })
+  name: string;
+
+  @Prop()
+  shortCode: string;
+
+  @Prop()
+  imgPath: string;
+
+  @Prop()
+  founded: number;
+
+  @Prop({
+    type: [{
+      id: { type: Number, required: true },
+      seasonId: { type: Number, required: true }
+    }],
+  })
+  statistics: [{
```

Comment:
Please go to [Player statistics field](https://github.com/AMRRAGHEB118/football_analytics_dashbaord_backend/blob/1707a657f5ef881b1f952f7b3b8ecba4040ae36e/src/player/schema/player.schema.ts#L49)